rust/example-borrow/src/main.rs

```diff
@@ -0,0 +1,55 @@
+// Example file for borrowing and move semantik
+
+// Show a simple reassign
+fn simple_reassign() {
+    let var1 = 1;
+    let var2 = var1;
+
+    println!("Die Werte sind: {} und {}", var1, var2);
+}
+
+// Show a simple reassign
+fn simple_string_reassign() {
+    let var1 = "Foo"; // var1 is a reference here
+    // This would not work
+    // let var1 = String::from("hello");
+    let var2 = var1;
```

Comment:
Ohne die Information dass es in Rust String und str gibt, ist dieses Beispiel nur schwer verständlich.
Der IDE würde zwar den Typ anzeigen, aber ich denke den Typ explizit anzugeben, würde die Lesbarkeit auch direkt im Github erhöhen.